packages/merlin/merlin.3.4.0/opam

```diff
@@ -0,0 +1,74 @@
+opam-version: "2.0"
+maintainer:   "defree@gmail.com"
+authors:      "The Merlin team"
+homepage:     "https://github.com/ocaml/merlin"
+bug-reports:  "https://github.com/ocaml/merlin/issues"
+dev-repo: "git+https://github.com/ocaml/merlin.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" "1"] {with-test & ocaml:version >= "4.03"}
+]
+depends: [
+  "ocaml" {>= "4.02.3" & < "4.12"}
+  "dune" {>= "1.8.0"}
+  "dot-merlin-reader" {= "3.4.0"}
```

Comment:
Note for next time that this can be replaced by `{= version}`. This is probably gonna be easier to maintain this way (unless you're planning to release both separately).

Reply:
We did not decide yet if we are going to release separately or not.
Hopefully `dot-merlin-reader` should change very little...